pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -184,6 +184,9 @@ public static class ControllerPeriodicTasksConf {
         "controller.realtimeSegmentRelocation.initialDelayInSeconds";
     public static final String SEGMENT_RELOCATOR_INITIAL_DELAY_IN_SECONDS =
         "controller.segmentRelocator.initialDelayInSeconds";
+    public static final String SEGMENT_RELOCATOR_ENABLE_LOCAL_TIER_MIGRATION =
```

Comment:
Why do we need a separate config for this? Isn't this the same as tier assigner?

Reply:
SegmentTierAssigner just calculates target tiers for segments, but not trigger the real migration.

The SegmentRelocator is extended to kick off the tier migration automatically. But it can be turned off via this knob, if someone wants to trigger this manually to fully control when the migration starts.